src/store/root.store.ts

```diff
@@ -132,10 +132,10 @@ export default class RootStore {
 
   @action
   getUserLoginDetails() {
-    fetchUserLoginStatus()
+    fetchUserInfo()
       .then((userData) => {
-        this.isUserAuthenticated = userData.authenticated;
-        this.userName = userData.userName;
+        this.userInfo.firstName = userData.email;
```

Comment:
why do we choose email as user name, also we show a hello msg as an email?

Reply:
fixed to user first name, this was an initial setup. since the information retrieved from the server may not include name and email is the unique identifier.